pkg/backup/backup_compaction_test.go

```diff
@@ -270,6 +271,48 @@ crdb_internal.json_to_pb(
 			t, db, []string{"foo"}, "'nodelocal://1/backup'", opts,
 		)
 	})
+
+	t.Run("pause resume and cancel", func(t *testing.T) {
+		db.Exec(t, "CREATE TABLE foo (a INT, b INT)")
+		defer func() {
+			db.Exec(t, "DROP TABLE foo, bar")
+		}()
+		db.Exec(t, "INSERT INTO foo VALUES (1, 1)")
+		start := getTime()
+		db.Exec(t, fmt.Sprintf(fullBackupAostCmd, start))
+		db.Exec(t, "INSERT INTO foo VALUES (2, 2)")
+		db.Exec(t, incBackupCmd)
+		db.Exec(t, "CREATE TABLE bar (a INT, b INT)")
+		db.Exec(t, "INSERT INTO bar VALUES (1, 1)")
+		db.Exec(t, incBackupCmd)
+		db.Exec(t, "INSERT INTO bar VALUES (2, 2)")
+		db.Exec(t, incBackupCmd)
+		db.Exec(t, "INSERT INTO bar VALUES (3, 3)")
+		end := getTime()
+		db.Exec(t, fmt.Sprintf(incBackupAostCmd, end))
+		db.Exec(t, "SET CLUSTER SETTING jobs.debug.pausepoints = 'backup.after.details_has_checkpoint'")
+		defer func() {
+			db.Exec(t, "SET CLUSTER SETTING jobs.debug.pausepoints = ''")
+		}()
+
+		jobID := startCompaction(start, end)
+		jobutils.WaitForJobToPause(t, db, jobID)
+		db.Exec(t, "RESUME JOB $1", jobID)
+		waitForSuccessfulJob(t, tc, jobID)
+		validateCompactedBackupForTables(t, db, []string{"foo", "bar"}, "'nodelocal://1/backup'")
+
+		db.Exec(t, "SET CLUSTER SETTING jobs.debug.pausepoints = ''")
+		db.Exec(t, "INSERT INTO bar VALUES (4, 4)")
+		db.Exec(t, incBackupCmd)
```

Comment:
is this extra vanilla inc backup necessary?

Reply:
Nope, had it for the same reason above.